testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -396,7 +396,13 @@
 Given(/^I am authorized for the "([^"]*)" section$/) do |section|
   case section
   when 'Admin'
-    step %(I am authorized as "admin" with password "admin")
+    if $username
+      step %(I am authorized as "#{$username}" with password "#{$password}")
+      step %(I am logged in API as user "#{$username}" and password "#{$password}")
```

Comment:
isn't that redundant?

Reply:
Not really, one is log-in from the WebUI so in the browser.
The other is log-in from the API, so keeping a connection established in our api_client

Reply:
ah yes of course

but then i would do the same in case 'Images' just below, for consistency

Reply:
I think kiwikiwi has some special permissions different than admin, also we are using it in very specific test cases.
So for now, better if we don't touch it...